oak/server/rust/oak_runtime/src/lib.rs

```diff
@@ -432,16 +438,46 @@ impl OakRuntime {
     pub fn node_create(
         &mut self,
         node_name: String,
-        config: &str,
+        config_name: &str,
+        entrypoint_name: &str,
         handle: Handle,
     ) -> Result<NodeStartInfo, OakStatus> {
-        // First, find the code referred to by the config name.
-        let entrypoint = match self.entrypoint(&config) {
-            Some(e) => e,
+        // First, check the config name exists.
+        let config = match self.node_configs.get(config_name) {
```

Comment:
(optional)

```rust
let config = self.node_configs.get(config_name).unwrap_or(|| {
  debug!(...);
  Err(...)
})?;
```

(though it probably needs an extra `.map(Result::Ok)` somewhere).